test/Microsoft.Azure.WebJobs.Extensions.Sql.Tests.csproj

```diff
@@ -19,10 +19,25 @@
 
   <Target Name="CopySamples" AfterTargets="Build">
     <ItemGroup>
-      <_CopyItems Include="..\samples\samples-csharp\bin\$(Configuration)\$(TargetFramework)\**\*.*" />
+      <_CSharpCopyItems Include="..\samples\samples-csharp\bin\$(Configuration)\$(TargetFramework)\**\*.*" />
     </ItemGroup>
-    <Copy SourceFiles="@(_CopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\CSharp\%(RecursiveDir)" />
+    <Copy SourceFiles="@(_CSharpCopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\CSharp\%(RecursiveDir)" />
     <Message Text="Copied Samples output to $(OutDir)\SqlExtensionSamples" Importance="high" />
+    <ItemGroup>
+      <_JsCopyItems Include="..\samples\samples-js\**\*.*" />
+    </ItemGroup>
+    <Copy SourceFiles="@(_JsCopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\JavaScript\%(RecursiveDir)" />
+    <Message Text="Copied JS Samples output to $(OutDir)\samples-js" Importance="high" />
+    <ItemGroup>
+      <_HostCopyItems Include="host.json" />
+    </ItemGroup>
+    <Copy SourceFiles="@(_HostCopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\JavaScript\" />
+    <Message Text="Copied host.json to $(OutDir)\SqlExtensionSamples\JavaScript" Importance="high" />
+    <ItemGroup>
+      <_JsCopyItems Include="Integration\test-js\**\*.*" />
```

Comment:
Did you try combining these into a single ItemGroup with a single copy? Right now I believe you'll be copying the samples-js content multiple times since you're just adding items to the _JsCopyItems list.

Reply:
Wasn't sure if I can add more than one copy items in an itemGroup, will try, thanks